### PR TITLE
update UIView shadow Extension

### PR DESCRIPTION
**Pull request by embassem**
## Checklist

- [ ] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [ ] New extensions are written in Swift 4.
- [ ] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [ ] I have added tests for new extensions, and they passed.
- [ ] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [ ] All extensions are declared as **public**.
- [ ] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

**Comment by SD10** [comment]
@embassem We also need to add unit tests for any new extensions.

File: Sources/Extensions/UIKit/UIViewExtensions.swift [hunk 1]
```diff
@@ -79,10 +79,19 @@ public extension UIView {
 			return layer.cornerRadius
 		}
 		set {
-			layer.masksToBounds = true
 			layer.cornerRadius = abs(CGFloat(Int(newValue * 100)) / 100)
 		}
 	}
+    
+    /// SwifterSwift: masksToBounds to layer of view; also inspectable from Storyboard.
+    @IBInspectable public var masksToBounds: Bool {
```
Comment (guykogus) [inline Sources/Extensions/UIKit/UIViewExtensions.swift]: Please add a simple test for this.

**Comment by SD10** [comment]
@embassem Any update on the requested changes?

**Comment by SD10** [comment]
@embassem I'm going to close this one due to inactivity. Please feel free to reopen with the requested changes 👍

**Pull request closed without merging**